Softeq.XToolkit.WhiteLabel/Model/PagingModel.cs

```diff
@@ -5,12 +5,35 @@
 
 namespace Softeq.XToolkit.WhiteLabel.Model
 {
+    /// <summary>
+    ///    A model for representing data pagination.
+    /// </summary>
+    /// <typeparam name="T">The type of record.</typeparam>
     public class PagingModel<T>
     {
+        /// <summary>
+        ///     Gets or sets page data (list of records).
+        /// </summary>
         public IList<T> Data { get; set; } = default!;
+
+        /// <summary>
+        ///     Gets or sets the number of the current page.
```

Comment:
The word "number" sounds unnatural in this context. Maybe it's better to use "index" instead?